packages/paste-website/src/pages/components/modal/index.mdx

```diff
@@ -0,0 +1,423 @@
+---
+
+import {graphql} from 'gatsby';
+import {useUID} from 'react-uid';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Button} from '@twilio-paste/button';
+import {Grid, Column} from '@twilio-paste/grid';
+import {Modal, ModalBody, ModalFooter, ModalFooterActions, ModalHeader, ModalHeading} from '@twilio-paste/modal';
+import {FormLabel, FormInput} from '@twilio-paste/form'
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Codeblock} from '../../../components/codeblock';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {defaultExample, initialFocusExample, wideExample} from '../../../component-examples/ModalExamples'
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/modal"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/modal/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/modal"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-list--ordered-list"
+  abstractUrl="https://share.goabstract.com/85c81258-90b9-49c8-b34b-9418da5f2f00"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Modals
+
+A Modal is a dialog that appears on top of the main content and moves the system into a special mode requiring user interaction. Modals are typically used to command a user's attention for any of these scenarios:
+
+- When you want to capture information from the user without having them leave the parent page.
+- When you want to show additional information to the user without losing context of the parent page.
+
+If you need to *interrupt* a user's current task to catch their full attention, especially if you need to warn them to prevent an error, use an Alert Dialog instead (component to come). An Alert Dialog ensures the user acknowledges the information in the dialog but is visually similar to a Modal.
+
+Users cannot interact with content outside an active modal window until the user explicitly interacts with it. When the modal opens, focus moves to an element contained in it, and the focus should stay in and cycle through the modal's content. Focus shouldn't return to the underlying page until the user:
+
+- Presses the Esc key
+- Presses the close "x" button in the Modal header,
```

Comment:
remove the , after the list and the oxford comma before the second last list item